### ISSUE TITLE
Website takes pre-order placeholder into accounts when computing platforms

### ISSUE THREAD

**Issue by fasterthanlime** [itchio/itch.io#895]
https://github.com/itchio/itch/issues/1242

**Comment by leafo** [comment]
the p_windows, etc. fields were designed for browsing classification. I think it makes sense to show those games in the Windows section if they have a windows version that will be available. We could alter the api to change the meaning of those fields just when returning a game object, but I'm unsure that's the best approach.

Can the app be aware of pre-order files and tell the user what's going on or be more explicit when trying to interact with a page that has them?

**Comment by fasterthanlime** [comment]
> We could alter the api to change the meaning of those fields just when returning a game object, but I'm unsure that's the best approach.

Yeah, I was going to say, I think it makes sense to have those be separate.

> Can the app be aware of pre-order files and tell the user what's going on or be more explicit when trying to interact with a page that has them?

tl;dr - this is going to N+1. The install button here:

![image](https://user-images.githubusercontent.com/7998310/45671853-e5010200-bb26-11e8-90b3-488dd101cb82.png)

is the same button as here:

![image](https://user-images.githubusercontent.com/7998310/45671887-f5b17800-bb26-11e8-8e7e-496f784f3a2e.png)

they both interpret information from:

  * The game API response
  * All the profile owned keys (which are stored globally in the application's state)

I can make that button automagically fetch game uploads (and even have butler cache them) - but that means whenever loading up a collection, it'll fire up 15 API requests for game uploads (more if it's restoring scroll), just to know whether the game can be installed or not. I'm not sure what's the best way to go here.

in current v25, when you try to install a game that only has pre-order uploads, you see this:

![image](https://user-images.githubusercontent.com/7998310/45672001-49bc5c80-bb27-11e8-9047-9cd78085ede5.png)

I think it should probably just show a message like "nothing is available to install at the moment", with a close button instead of an install button.
